### PR TITLE
1000 location saving

### PR DESCRIPTION
**Pull request by tibetsprague**
Fixes https://github.com/Hylozoic/hylo-evo/issues/1000

Maybe fixes https://github.com/Hylozoic/hylo-evo/issues/956 ?

### CODE REVIEW

File: src/components/Map/layers/clusterLayer/index.js [hunk 1]
```diff
@@ -144,7 +144,7 @@ export default class PostClusterLayer extends CompositeLayer {
 
           return d.properties.type + '.png'
         },
-        getSize: d => d.properties.cluster ? 50 : 22,
+        getSize: d => d.properties.cluster ? 34 : 15,
```
Comment (thomasgwatson) [inline src/components/Map/layers/clusterLayer/index.js:147]: Are these little styling tweaks, while you were looking at location stuff?

File: src/routes/GroupSettings/GroupSettings.connector.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ export function mapStateToProps (state, props) {
   return {
     canModerate,
     currentUser,
+    fetchPending: state.pending[FETCH_GROUP_SETTINGS],
```
Comment (thomasgwatson) [inline src/routes/GroupSettings/GroupSettings.connector.js:25]: Is this related?

Comment (thomasgwatson) [reply src/routes/GroupSettings/GroupSettings.connector.js:25]: So, related to a group's location?

Comment (tibetsprague) [reply src/routes/GroupSettings/GroupSettings.connector.js:25]: yes, part of why it was getting reset is the state was getting reset in the component all the time. this solved it

File: src/components/Map/layers/clusterLayer/index.js (hunk 1 above)
Comment (tibetsprague) [reply src/components/Map/layers/clusterLayer/index.js:147]: yeah, unrelated. https://github.com/Hylozoic/hylo-evo/issues/956

File: src/routes/GroupSettings/GroupSettingsTab/GroupSettingsTab.js [hunk 3]
```diff
@@ -97,12 +99,12 @@ export default class GroupSettingsTab extends Component {
   save = async () => {
     this.setState({ changed: false })
     const { group, fetchLocation } = this.props
-    let coordinateLocationId
+    let locationId = this.state.edits.locationId
     if (group && this.state.edits.location !== group.location) {
-      coordinateLocationId = await ensureLocationIdIfCoordinate({ fetchLocation, location: this.state.edits.location, locationId: this.state.edits.locationId })
+      locationId = await ensureLocationIdIfCoordinate({ fetchLocation, location: this.state.edits.location, locationId })
```
Comment (thomasgwatson) [inline src/routes/GroupSettings/GroupSettingsTab/GroupSettingsTab.js:104]: Do these changes impact the function of the code, or does it just rearrange variables in a cosmetic way?

Comment (tibetsprague) [reply src/routes/GroupSettings/GroupSettingsTab/GroupSettingsTab.js:104]: just cleanup as I was playing around with it

**Review by thomasgwatson — APPROVED** [review]
